source/features/filter-pr-by-build-status.tsx

```diff
@@ -41,6 +41,11 @@ function populateDropDown({currentTarget}: Event): void {
 }
 
 function init(): void {
+	// If there are no PRs then return
+	if (!select('.table-list-filters')) {
```

Comment:
Prefer `select.exists`. We are following the same everywhere.

Reply:
Duh, I'll fix it.